res/skins/Traktmixxx/.gitignore

```diff
@@ -0,0 +1,16 @@
+RoundedCorners.sublime-project
```

Comment:
Please revert those changes (doing a force push) and instead extend your global .gitignore in your home directory.